howso/hierarchy.amlg

```diff
@@ -751,6 +751,177 @@
 		(call !Return (assoc payload (assoc "count" (size cases_to_move)) ))
 	)
 
+	;method to combine cases from all subtrainees into the root trainee and then deletes the subtrainees and removes them from the hierarchy
+	;{long_running (true) idempotent (true)}
+	#combine_with_subtrainees
+	(declare
+		;returns {ref "CombineResponse"}
+		(assoc
+			;{type "list"}
+			;optional list of subtrainee ids
+			subtrainees (null)
```

Comment:
Should this be named `child_ids` instead? Looks like we use `child_id` in the other subtrainee methods.